### PR TITLE
Use constant operand for modulo

### PR DESCRIPTION
**Pull request by sdkrystian**
Implements #115

### CODE REVIEW

**Comment by vinniefalco** [comment]
The commit message body should have the text `fix #115`

File: include/boost/json/detail/object_impl.hpp [hunk 1]
```diff
@@ -133,6 +135,21 @@ class object_impl
         index_t& head,
         value_type& p) noexcept;
 
+    //------------------------------------------------
+    // https://github.com/CPPAlliance/json/issues/115
```
Comment (vinniefalco) [inline include/boost/json/detail/object_impl.hpp:139]: dont need this comment

File: include/boost/json/detail/object_impl.hpp [hunk 2]
```diff
@@ -133,6 +135,21 @@ class object_impl
...
+    // 
+    static
+    inline
+    prime_list&
+    primes() noexcept;
+
+    static
+    inline
+    std::size_t
+    position(
```
Comment (vinniefalco) [inline include/boost/json/detail/object_impl.hpp:149]: need a better name for this

File: include/boost/json/detail/object_impl.hpp [hunk 3]
```diff
@@ -133,6 +135,21 @@ class object_impl
         index_t& head,
         value_type& p) noexcept;
 
+    //------------------------------------------------
+    // https://github.com/CPPAlliance/json/issues/115
+    // 
+    static
+    inline
+    prime_list&
+    primes() noexcept;
```
Comment (vinniefalco) [inline include/boost/json/detail/object_impl.hpp:144]: should this be called `bucket_sizes`?

File: include/boost/json/detail/object_impl.hpp (hunk 2 above)
Comment (sdkrystian) [reply include/boost/json/detail/object_impl.hpp:149]: `bucket_index`?

**Comment by vinniefalco** [comment]
These changes look good but we need to explain the benchmark results.

**Comment by sdkrystian** [comment]
@vinniefalco Explain in what way? Like a comment as to why this is better?

**Pull request closed without merging**